tasks/src/zero-shot-classification/about.md

```diff
@@ -0,0 +1,20 @@
+# Zero Shot Classification
+
+## About the Task
+
+Zero Shot Classification ([ZSC](https://joeddav.github.io/blog/2020/05/29/ZSL.html)) is a subset of [transfer learning](https://en.wikipedia.org/wiki/Transfer_learning) and an attribute of Large Language Models (LLM/s), effectively we use a large pretrained language model to generate a sequence of text that answers an informal question in the form of a prompt. The "prompt" is basically context for some task, [prompt engineering](https://en.wikipedia.org/wiki/Prompt_engineering) is a task in and of itself and involves coaxing the correct generations out of a LLM.
+- Zero Shot differs from Single-Shot or Few-Shot (learning/classification, both terms are used) as there are *no demonstrations* of the task explicitly presented in the prompt. For Single, Few or N -Shot classification a User provides several correct demonstrations of the desired task (the ideal mapping of inputs > outputs in the prompt) to the model. 
```

Comment:
The first sentence & parenthesis are creating a bit of an ambiguity. You can feel free to add as much sentences as you want, so long as they're easily readible 🙂 You can check out existing pages for language: hf.co/tasks